### PR TITLE
Apply ansible best practices (priority on owner:group:mode)

### LINKED ISSUE #546: When using tasks creating files on FS, we should indicate owner/group/mode
It's a good ansible pratice to set these attributes, moreover, it will make us not dependent on system umask

### PR DESCRIPTION
**Pull request by gboutry**
#### Which issue(s) this PR fixes

Fixes #546

#### Additional comments

:warning: Tell me if you'd rather have a PR by service instead of a big PR

- :warning: JMX Password are rendered as `root:root:644`
- This PR tries not to change big behaviors.
- Ranger are chowned to `{ranger_user}:{hadoop_group}` for idempotence

Here are the remaining warnings that would need dedicate PRs:
```yaml
  - command-instead-of-shell  # Use shell only when shell functionality is required
  - deprecated-command-syntax  # Using command rather than an argument to e.g. file
  - experimental  # all rules tagged as experimental
  - no-changed-when  # Commands should not change things if nothing needs doing
  - risky-shell-pipe  # Shells that use pipes should set the pipefail option
  - yaml  # Violations reported by yamllint
```
#### Agreements

- [X] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [X] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Comment by gboutry** [comment]
Force push is rebase on master

**Review by mehdibn — APPROVED** [review]

**Review by mehdibn — APPROVED** [review]

**Review by rpignolet — CHANGES_REQUESTED** [review]

File: roles/hive/hiveserver2/tasks/jmx-exporter.yml [hunk 1]
```diff
@@ -1,13 +1,19 @@
-# Copyright 2022 TOSIT.IO
+# Copyright 2022 TOSIT.IO755
```
[the remark this answers is not in the document]
Comment (gboutry) [reply roles/hive/hiveserver2/tasks/jmx-exporter.yml]: Solved

File: roles/utils/ssl_tls/tasks/convert_cert_key.yml [hunk 2]
```diff
@@ -21,4 +21,4 @@
     path: "{{ certs_dir }}/{{ ansible_fqdn }}.p12"
     owner: root
     group: root
-    mode: "0600"
+    mode: 0600
```
[the remark this answers is not in the document]
Comment (gboutry) [reply roles/utils/ssl_tls/tasks/convert_cert_key.yml]: Solved in last force pushed

File: roles/utils/ssl_tls/tasks/create_keystore.yml [hunk 3]
```diff
@@ -58,4 +59,4 @@
     path: "{{ keystore_location }}"
     owner: root
     group: "{{ hadoop_group }}"
-    mode: "0640"
+    mode: 0640
```
[the remark this answers is not in the document]
Comment (gboutry) [reply roles/utils/ssl_tls/tasks/create_keystore.yml]: Solved in last force pushed

File: roles/utils/ssl_tls/tasks/create_truststore.yml [hunk 4]
```diff
@@ -19,4 +19,4 @@
     path: "{{ truststore_location }}"
     owner: root
     group: root
-    mode: "0644"
+    mode: 0644
```
[the remark this answers is not in the document]
Comment (gboutry) [reply roles/utils/ssl_tls/tasks/create_truststore.yml]: Solved in last force pushed

**Review by rpignolet — APPROVED** [review]

**Review by leopaul36 — APPROVED** [review]
Validated with a fresh `tdp-getting-started` install

**Pull request merged**